packages/neo-one-monitor/src/types.js

```diff
@@ -85,6 +86,15 @@ export type LogSingleOptions = {|
   },
 |};
 
+export type LoggerLogOptions = {|
```

Comment:
This should stay in `MonitorBase`, it's not part of the public api of `Monitor`.

Reply:
It's ok to import this from `MonitorBase` directly where you need it.